src/storage/storage_backend.rs

```diff
@@ -160,6 +160,11 @@ pub trait StorageBackend<U: Sync + Send + Debug>: Send + Sync + Debug {
     /// [`Metadata`]: ./trait.Metadata.html
     async fn metadata<P: AsRef<Path> + Send + Debug>(&self, user: &Option<U>, path: P) -> Result<Self::Metadata>;
 
+    /// Returns the `Md5` for the given file.
+    ///
+    /// [`Metadata`]: ./trait.Md5.html
```

Comment:
Isn't this a link reference. But its not used?

Reply:
yeah that doesn't belong there